ixofoundation/ixo-webclient
# chore: update entity header

## Scope
https://nonaredteam.atlassian.net/browse/IXO-596

## Work Done
Updated the styling of the section
Added the JSON functionality

## Steps to Test
Run the app and make sure the styling matches the design and all the acceptance criteria in the task is met

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/84049503-d0c10080-a9ac-11ea-9713-b28b5849a20e.png)
